### DISCUSSION TITLE
Nested folder access error with argo workflow

### DISCUSSION THREAD

**Discussion by aishwaryagirish** [argoproj/argo-workflows#10836] [General]
I am having an issue parsing through nested folder with argo workflow inside azure storage. I wanted to know if it is a known issue and how to fix it. This is the error I see

artifact testash2 failed to load: unable to download directory /models/SKLearnServer/1680718691765417800/workspace/code: unable to download file /argo/inputs/artifacts/testash2.tmp/model/metadata.yaml: unable to create file /argo/inputs/artifacts/testash2.tmp/model/metadata.yaml: open /argo/inputs/artifacts/testash2.tmp/model/metadata.yaml: not a directory

**Comment by viploveb** [comment]
I am facing the same error, created an issue for the same. #11385
